harness/asyncHelpers.js

```diff
@@ -50,14 +50,17 @@ function asyncTest(testFunc) {
  */
 assert.throwsAsync = function (expectedErrorConstructor, func, message) {
   return new Promise(function (resolve) {
-    var expectedName = expectedErrorConstructor.name;
-    var expectation = "Expected a " + expectedName + " to be thrown asynchronously";
     var fail = function (detail) {
       if (message === undefined) {
         throw new Test262Error(detail);
       }
       throw new Test262Error(message + " " + detail);
     };
+    if (typeof res !== "function") {
```

Comment:
`res` is not defined here

Reply:
Oops, forgot to change the variable name. Should be good now.